qiita_pet/test/test_study_handlers.py

```diff
@@ -382,5 +382,36 @@ class TestEBISubmitHandler(TestHandlerBase):
     pass
 
 
+class TestDelete(TestHandlerBase):
+    def test_delete_sample_template(self):
+        response = self.post('/study/description/1',
+                             {'sample_template_id': 1,
+                              'action': 'delete_sample_template'})
+        self.assertEqual(response.code, 200)
```

Comment:
Oh no, this isn't to make sure the delete function works, just that the delete page actually calls delete and the items are actually deleted when you hit the buttons on the page.